test/specs/photos-timeline.spec.js

```diff
@@ -0,0 +1,25 @@
+/* global describe, it, browser */
+const assert = require('chai').assert
+
+browser.timeouts('script', 60000)
+
+const assertVisualMatch = (result, title) => {
+  assert(result.isWithinMisMatchTolerance, `visual regression for ${title}`)
+}
+
+describe('Photos', function () {
+  it('loads the timeline', function () {
+    browser.url('cozy.local:8080')
```

Comment:
In my opinion : at build on Travis AND before making a new PR. On http://recette.cozycloud.cc, it is too late to detect regressions IMHO.

Also, it allows us to make functional tests locally. Priceless.